diff -Naur evdi-1.14.5.old/module/Makefile evdi-1.14.5/module/Makefile
--- evdi-1.14.5.old/module/Makefile	2024-07-14 14:14:15.751723722 +0200
+++ evdi-1.14.5/module/Makefile	2024-07-14 14:43:41.236192384 +0200
@@ -21,8 +21,6 @@
 RPIFLAG := -DRPI
 endif
 
-ifneq ($(DKMS_BUILD),)
-
 # DKMS
 
 KERN_DIR := /lib/modules/$(KERNELRELEASE)/build
@@ -40,69 +38,3 @@
 clean:
 	@echo $(KERN_DIR)
 	$(MAKE) KBUILD_VERBOSE=$(KBUILD_VERBOSE) M=$(CURDIR) SUBDIRS=$(CURDIR) SRCROOT=$(CURDIR) -C $(KERN_DIR) clean
-
-else
-
-# Not DKMS
-
-ifneq ($(KERNELRELEASE),)
-
-# inside kbuild
-# Note: this can be removed once it is in kernel tree and Kconfig is properly used
-ccflags-y := -isystem include/uapi/drm $(CFLAGS) $(EL8FLAG) $(EL9FLAG) $(RPIFLAG)
-evdi-y := evdi_platform_drv.o evdi_platform_dev.o evdi_sysfs.o evdi_modeset.o evdi_connector.o evdi_encoder.o evdi_drm_drv.o evdi_fb.o evdi_gem.o evdi_painter.o evdi_params.o evdi_cursor.o evdi_debug.o evdi_i2c.o
-evdi-$(CONFIG_COMPAT) += evdi_ioc32.o
-obj-$(CONFIG_DRM_EVDI) := evdi.o
-obj-y += tests/
-
-else
-
-# kbuild against specified or current kernel
-CP ?= cp
-DKMS ?= dkms
-RM ?= rm
-
-MODVER=1.14.5
-
-ifeq ($(KVER),)
-	KVER := $(shell uname -r)
-endif
-
-ifneq ($(RUN_DEPMOD),)
-	DEPMOD := /sbin/depmod -a
-else
-	DEPMOD := true
-endif
-
-ifeq ($(KDIR),)
-	KDIR := /lib/modules/$(KVER)/build
-endif
-
-MOD_KERNEL_PATH := /kernel/drivers/gpu/drm/evdi
-
-default: module
-
-module:
-	$(MAKE) -C $(KDIR) M=$$PWD
-
-clean:
-	$(RM) -rf *.o *.a *.ko .tmp* .*.*.cmd Module.symvers evdi.mod.c modules.order
-
-install:
-	$(MAKE) -C $(KDIR) M=$$PWD INSTALL_MOD_PATH=$(DESTDIR) INSTALL_MOD_DIR=$(MOD_KERNEL_PATH) modules_install
-	$(DEPMOD)
-
-uninstall:
-	$(RM) -rf $(DESTDIR)/lib/modules/$(KVER)/$(MOD_KERNEL_PATH)
-	$(DEPMOD)
-
-install_dkms:
-	$(DKMS) install .
-
-uninstall_dkms:
-	$(DKMS) remove evdi/$(MODVER) --all
-	$(RM) -rf /usr/src/evdi-$(MODVER)
-
-endif # ifneq ($(KERNELRELEASE),)
-
-endif # ifneq ($(DKMS_BUILD),)
